common/history/pops/11_east_asia.txt

```diff
@@ -379,40 +379,64 @@
 		region_state:MCH = {
 			create_pop = {
 				culture = manchu
-				size = 5210800
+				size = 1210800
 			}
 			create_pop = {
 				culture = han # nanfaren
-				size = 2511992
+				size = 411992
 			}
 			create_pop = {
 				culture = han # nanfaren
-				size = 511992
+				size = 100000
 				religion = mahayana
 			}			
 			create_pop = {
 				culture = korean
-				size = 504000
+				size = 54000
 			}
 			create_pop = {
 				culture = mongol
-				size = 80412
+				size = 120412
+			}
+		}
+	}
+	s:STATE_GANDO = {
+		region_state:MCH = {
+			create_pop = {
+				culture = manchu
+				size = 89752
+			}
+			create_pop = {
+				culture = han
+				size = 50120
+			}
+			create_pop = {
+				culture = korean
+				size = 295272
+			}
+			create_pop = {
+				culture = siberian
+				size = 5000
 			}
 		}
 	}
 	s:STATE_NORTHERN_MANCHURIA = {
 		region_state:MCH = {
 			create_pop = {
 				culture = manchu
-				size = 2821200
+				size = 400000
```

Comment:
Not to nitpick, but I don't enjoy the roundness of these numbers. Try adding a bit of variation to each of them.